rebar.config

```diff
@@ -33,12 +33,15 @@
 
 {escript_emu_args, "%%! -connect_all false\n" }.
 {escript_incl_extra, [{"erlang_ls/priv/snippets/*", "_build/default/lib/"}]}.
+{escript_main_app, erlang_ls}.
```

Comment:
Found this in the rebar3 docs and I was planning to do exactly the same. Good stuff!

Reply:
This let's us also change erlang_ls app to els_lsp